### ISSUE TITLE
Can't create subaccount with known seed phrase

### ISSUE THREAD

**Issue by mikedotexe** [near/near-cli#1096]
**Describe the bug**
I'd like to make wc.mike.near because I need to test WalletConnect on a few mobile apps and don't want to give my seed phrase for mike.near. I have downloaded NEAR Mobile, and it's asking me for a seed phrase. 

**To Reproduce**
Try creating a subaccount with a seed phrase you've chosen

**Expected behavior**
We used to be able to specify the seed phrase for a new subaccount using NEAR CLI

**Comment by VlodkoMr** [comment]
The same issue with near-cli call. Example:

```
 ACCOUNT_ID=test-dao.near
 CONTRACT=v1.test-dao.near
 NEAR_ENV=mainnet near create-account "$CONTRACT" --masterAccount "$ACCOUNT_ID" --initialBalance 5
```
Account test-dao.near exists on mainnet and funded, also loginned in cli.

But I get error:
**`TypedError: Can't complete the action because account v1.test-dao.near doesn't exist`**

**Comment by gagdiez** [comment]
@VlodkoMr 

`NEAR_ENV=mainnet yarn start create-account "$contract" --useAccount "$account_id" --initialBalance 0.2` worked perfectly for me on MacOS, which environment are you using?

**Comment by gagdiez** [comment]
@mikedotexe this is back in version `4.0.8`

**Comment by gagdiez** [comment]
@VlodkoMr your error was also reported in #1106 and fixed by #1108
